.changes/v3.3.0/673-bug-fixes.md

```diff
@@ -0,0 +1,2 @@
+* (Issue #634) Setting CPU count to be less than template has throws error. [GH-673]
```

Comment:
Grammar suggestion and shouldn't the `(Issue #634)` go to the end as GH-634 for auto-parsing during release? We have some changelog entries with multiple links already.
```suggestion
* (Issue #634) Setting CPU count to less than what the template has throws error [GH-673]
```

Reply:
Fixed.